src/main/schemas/territories.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "http://isdcf.com/ns/json-schemas/registries/territories/1.0.0-beta.1",
+  "$comment": "Copyright, ISDCF <info@isdcf.com>",
+  "title": "Schema for the Territories Registry of the ISDCF",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "comments": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "name": {
+        "type": "string",
+        "pattern": "^[a-zA-Z ']+$"
+      },
+      "dcncTag": {
+        "type": "string",
+        "pattern": "^[A-Z0-9]{2}$"
+      },
+      "tag": {
```

Comment:
change to "rfc5646Tag"

Reply:
The tag does not always conform to RFC 5646, e.g. `AE-DU`, so the name `tag` seems more appropriate.

Reply:
Only Dubai and Kosovo are in this situation correct? Should we have 2 fields then? "rfc5646Tag" for those that conform, and "tag", which is governed by the tagScope?

Reply:
I do not see the need to qualify the symbol `tag` since the `tagScope` already qualifies it. In other words, no need to bake a namespace into `tag` since `tagScope` provides a namespace.

Reply:
I see, that makes sense. Fair enough.